src/resource/solidDataset.ts

```diff
@@ -538,6 +565,39 @@ export async function createContainerInContainer(
   return resourceWithResourceInfo;
 }
 
+/**
+ * Deletes the Container at a given URL.
+ *
+ * @param file The (URL of the) Container to delete
+ */
+export async function deleteContainer(
+  container: Url | UrlString | WithResourceInfo,
+  options: Partial<
+    typeof internal_defaultFetchOptions
+  > = internal_defaultFetchOptions
+): Promise<void> {
+  const url = hasResourceInfo(container)
+    ? internal_toIriString(getSourceUrl(container))
+    : internal_toIriString(container);
+  if (!url.endsWith("/")) {
```

Comment:
Shouldn't this use `isContainer` ? Possibly, we could create a new `isContainerUrl` function that would be called by `isContainer`, in order to take a `Url | UrlString` as a parameter instead of a `WithResourceInfo`.

Reply:
Yes: e0a107f8c66c7e1c4b5c2cc3f1afb6368e8bdca0